server/tests/test_notebook_revisions_cleanup.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+from unittest.mock import patch
+
+import pytest
+import pytz
+from django.urls import reverse
+
+from server.notebooks.models import Notebook, NotebookRevision
+from server.notebooks.tasks import execute_notebook_revisions_cleanup
+
+# sorted by time asc
+CREATED_DATETIMES = [
+    # 10:13
+    datetime(2019, 11, 20, 10, 13, 10, tzinfo=pytz.utc),
+    datetime(2019, 11, 20, 10, 13, 40, tzinfo=pytz.utc),
+    # 10:14
+    datetime(2019, 11, 20, 10, 14, 10, tzinfo=pytz.utc),
+    datetime(2019, 11, 20, 10, 14, 30, tzinfo=pytz.utc),
+    datetime(2019, 11, 20, 10, 14, 40, tzinfo=pytz.utc),
+    # 10:15
+    datetime(2019, 11, 20, 10, 15, 10, tzinfo=pytz.utc),
+    datetime(2019, 11, 20, 10, 15, 40, tzinfo=pytz.utc),
+]
+NOW_UTC = datetime(2019, 11, 20, 10, 15, 50, tzinfo=pytz.utc)
+
+
+@pytest.fixture
+def notebook_and_draft_revisions(fake_user):
+    notebook = Notebook.objects.create(owner=fake_user, title="Fake notebook")
+    notebook_revisions = []
+
+    for i, created in enumerate(CREATED_DATETIMES):
+        revision = NotebookRevision.objects.create(
+            notebook=notebook,
+            title="Revision %s" % i,
+            content="fake notebook content for revision %s" % i,
```

Comment:
```suggestion
            content=f"fake notebook content for revision {i}",
```

Reply:
I used `f-string`, but current lint complained about it. I didn't plan to fix this lint problem in this PR.